icons/unity.svg

```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img"><title>Unity icon</title>
```

Comment:
The SVG seems to not fill its viewbox entirely - there is a bit of whitespace around it (especially to the left of it).

The SVG itself should also be on one line. There's currently a newline after `</title>`